components/admin_console/team_channel_settings/channel/details/channel_moderation.test.tsx

```diff
@@ -210,6 +215,20 @@ describe('admin_console/team_channel_settings/channel/ChannelModeration', () =>
                 onChannelPermissionsChanged={onChannelPermissionsChanged}
                 teamSchemeID={teamSchemeID}
                 teamSchemeDisplayName={teamSchemeDisplayName}
+                guestAccountsEnabled={true}
+            />
+        );
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('Should match sixth Snapshot', () => {
+        const wrapper = shallow(
+            <ChannelModeration
+                channelPermissions={channelPermissions}
+                onChannelPermissionsChanged={onChannelPermissionsChanged}
+                teamSchemeID={undefined}
+                teamSchemeDisplayName={undefined}
+                guestAccountsEnabled={false}
```

Comment:
Disregard my comment above i see this now

Reply:
Reminded me to also add a snapshot for <ChannelModerationTableRow> with guestsAccountsEnabled = false to ensure the guests column does not render.